package/submariner-globalnet.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
When I deployed with this PR, some of the e2e tests are failing.


Summarizing 3 Failures:

[Fail] [dataplane] Basic TCP connectivity tests across clusters without discovery when a pod connects via TCP to a remote pod when the pod is on a gateway and the remote pod is not on a gateway [It] should have sent the expected data from the pod to the other pod 
/go/src/github.com/submariner-io/submariner/test/e2e/framework/framework.go:344

[Fail] [dataplane] Basic TCP connectivity tests across clusters without discovery when a pod connects via TCP to a remote service when the pod is not on a gateway and the remote service is on a gateway [It] should have sent the expected data from the pod to the other pod 
/go/src/github.com/submariner-io/submariner/test/e2e/framework/framework.go:344

[Fail] [dataplane] Basic TCP connectivity tests across clusters without discovery when a pod connects via TCP to a remote pod when the pod is on a gateway and the remote pod is on a gateway [It] should have sent the expected data from the pod to the other pod 
/go/src/github.com/submariner-io/submariner/test/e2e/framework/framework.go:344

Ran 10 of 10 Specs in 318.129 seconds
FAIL! -- 7 Passed | 3 Failed | 0 Pending | 0 Skipped
--- FAIL: TestE2E (318.13s)
FAIL
exit status 1
FAIL    github.com/submariner-io/submariner/test/e2e    318.146s
[submariner]$ ./scripts/e2e keep 1.14.2 false false helm false
FATA[0863] exit status 1                                
make: *** [Makefile:18: e2e] Error 1

Reply:
Okay. After switching to deploytool=operator, there are no errors and Submariner-globalnet is annotating the PODs/Services properly.